docs/source/whatsnew/pr/newfeat.rst

```diff
@@ -0,0 +1,14 @@
+YouTube video will now show thumbnail when exported to a media that do not support video. (PR #9086)
+
+Add warning when running `ipython <subcommand>` when subcommand is deprecated. `jupyter` should now be used.
+
+Code in `%pinfo` (also known as `??`) are now highlighter (:ghpull:`#8947`)
```

Comment:
Looking at the source of the github extension, I think these need to be without the `#`. I might be wrong,  though.